AdobeMediaGalleryApi/README.md

```diff
@@ -0,0 +1,9 @@
+# Magento_AdobeMediaGallery Module
```

Comment:
Please use the new common format for the readme. Can be taken from any other module in Adobe Stock Integration, i.e.: https://github.com/magento/adobe-stock-integration/blob/develop/AdobeIms/README.md